docs/ibexa_products/ibexa_commerce.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Ibexa Commerce
+description: Discover Ibexa Commerce and its essential features, capabilities and benefits.
+month_change: true
+---
+
+# Ibexa DXP Commerce edition product guide
+
+## What is [[= product_name_com =]]
+
+[[= product_name_com =]] is one of the edition offered by [[= product_name =]].
+This edition assists you in managing each aspect of the customer journey by combining content, customization, and commerce functions into a single, dedicated interface.
+[[= product_name_com =]] offers companies a streamlined, unified platform where they can personalize each component of the online shopping experience.
+You can completely revamp your online stores with and give your consumers exceptional purchasing experiences, from first contact to post-purchase support.
+
+![Ibexa Commerce](ibexa_commerce.png)
+
+## Availability
+
+To start using [[= product_name_com =]] you need to purchase a product license.
+For more information, see [[[= product_name_com =]] license pricing](https://www.ibexa.co/products/pricing?tab=3).
+You can also [contact us](https://www.ibexa.co/about-ibexa/contact-us) or [one of our partners](https://www.ibexa.co/partners).
+
+## How it works
+
+### Technical backstage
+
+With an active lincese, you can start [installation process](install_ibexa_dxp.md) that uses the Composer.
+
+[[= product_name_com =]] is based on [Symfony]([[= symfony_doc =]]). With a help of documentation and trainings, any developer familiar with Symfony or simply PHP may learn how to use available extension points and extend the platform.
+
+Version control systems and environment variables allow you to deploy your extensions and settings on several environments, such as [Ibexa Cloud](ibexa_cloud_guide.md).
+
+[[= product_name_com =]] is built on [[[= product_name_headless =]]](headless.md) and includes all bundles, APIs, and [features that comes with Headless edition](headless.md#core-features) as well as [Experience features](experience.md#core-features).
+
+## Capabilities and benefits
+
+[[= product_name_com =]] includes the greatest capabilities to accelerate your digital transformation: Personalized workflows, corporate account management, customized catalogs, and more.
+
+### Core features
+
+[[= product_name_com =]] includes all the features you need to launch your online store and reduce the time it takes to go live.
+
+#### Order management
```

Comment:
If we're mentioning order management - maybe we should mention Shipping and Payment management as well?